values.example.yaml

```diff
@@ -53,3 +53,195 @@ concourse:
     database: 'concourse'
```

Comment:
shouldn't this be values.yaml instead of example?
The whole idea of values.yaml is that you can overwrite the defaults